app/models/post.rb

```diff
@@ -0,0 +1,13 @@
+class Post < ApplicationRecord
+  belongs_to :user
+  has_many :comments
+  has_many :likes
+
+  def update_posts_counter
+    user.increment!(:postscounter)
+  end
```

Comment:
- [ ] [Optional] it will be nice to use the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - after_save to call the #update_post_counter method every time a new post has been saved to database, then make the update_post_counter method private. i.e
 ```
after_save :update_post_counter

  private

  def update_post_counter
    user.increment!(:postscounter)
  end
```

Reply:
Will update. Thank you.